src/shredder_flow/logic/VectorFieldGenerator.java

```diff
@@ -38,8 +40,37 @@ public void generateGradiantField(){
 		}
 	}
 
```

Comment:
Also please add another public method similar to `public void generateGradiantField()` named
`generateSymplecticVectorField()` that does exactly the same as generateGradiantField() but rotates the the vector by 90 degrees before setting it